rosdep/base.yaml

```diff
@@ -3762,6 +3762,10 @@ libfl-dev:
   fedora: [libfl-devel]
   opensuse: [libfl-devel]
   ubuntu: [libfl-dev]
+libflac-dev:
```

Comment:
Hello @Tomico-DEV. Thank you for helping make this package available to others in the ROS community! You might have noticed in your [previous pull request](https://github.com/ros/rosdistro/pull/46746/files that you got some suggestions from our auto-checker, suggesting you include keys for `rhel`, `opensuse` and `alpine`. Would you consider adding them too for us? Thank you! 

<img width="768" height="497" alt="Image" src="https://github.com/user-attachments/assets/69108622-84b1-4512-9583-ededca724446" />